src/python/turicreate/data_structures/sframe.py

```diff
@@ -1270,6 +1270,9 @@ def read_csv(cls,
             continue parsing skipping lines which fail to parse correctly.
             A sample of the first 10 encountered bad lines will be printed.
 
+        nrows_to_infer: integer
+            Get the number of nrows_to_infer rows enabling the arguments as specified.
```

Comment:
I don't really understand this description. I would just say something like: `The number of rows used to infer column types.`